### PR TITLE
updated UG with new note commands

### PR DESCRIPTION
**Pull request by Pinran-J**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -188,11 +188,39 @@ Clears all entries of people from the SectresBook.
...
 
-### Adding Notes `[coming in v1.2.1]`
+### Adding Notes `addNote`
 
-### Editing Notes `[coming in v1.2.1]`
+Adds a note to the SectresBook.
+
+Format: `addNote n_t/TITLE n_c/CONTENT `
+
+<div markdown="span" class="alert alert-primary">:bulb: **Tip:**
```
Comment (Neethesh26) [inline docs/UserGuide.md:197]: I think this line might be a bit weird, when I view it in the format, the 'Tip:' part is not bolded. Other than that looks good.

**Review by Neethesh26 — CHANGES_REQUESTED** [review]

File: docs/UserGuide.md (hunk 1 above)
Comment (Pinran-J) [reply docs/UserGuide.md:197]: image

I followed the same format as the 'add' command,

Comment (Pinran-J) [reply docs/UserGuide.md:197]: image

Should look like this in the page, i think its fine.
You can resolve this once you have confirmed. Thank you!

**Review by rui-han-crh — APPROVED** [review]
It looks okay to me

**Review by Neethesh26 — APPROVED** [review]

**Pull request merged**